docs/docs.md

```diff
@@ -514,12 +514,11 @@ Returns the rune for the passed in codepoint, `num`, which may be unicode, this
 
 ## clean(text)
 
-Strips any leading or trailing whitespace from `text`
+Strips any non-printable characters from `text`
```

Comment:
maybe we should have a separate `strip()` function that does what this was doing?